### PR TITLE
Adding an age column to the cli, closes #417

### LINKED ISSUE #417: Feature request: Display 'created' timestamp as "xxx minutes ago"
As an SRE, I deploy a lot of hcloud resources automatically via the hcloud API and use the cli to validate those changes. 
I am deploying using immutable VMs rather than phoenix deployments (instead of updating existing VMs, I re-create them with the newest config)
Since I am using the API in automation to create hcloud resources, my hcloud resources include Ids in their name.
To know which VM got deployed when, I often find myself using the hcloud-csi like this:

```bash
$ ❯ hcloud server list -s name -o 'columns=name,status,created,ipv4,ipv6'
NAME                           STATUS    CREATED                        IPV4             IPV6                   
cedi-dev-control-plane-xxxxx   running   Sun Nov 13 13:43:05 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-control-plane-xxxxx   running   Sun Nov 13 13:51:46 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-control-plane-xxxxx   running   Sun Nov 13 13:40:22 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-worker-cxp31-xxxxx    running   Wed Nov  9 17:15:32 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-worker-cxp31-xxxxx    running   Wed Nov  9 17:26:36 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-worker-cxp31-xxxxx    running   Wed Nov  9 16:46:55 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-worker-cxp31-xxxxx    running   Sun Nov 13 14:02:41 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
```

However, I noticed the `created` column contains the "raw" create DateTime string. 
Using the hcloud console in my browser, I can see the creating timestamp as a TimeDuration since now():

Screenshot 2022-11-13 at 14 17 18

With this ticket I want to gauge interest if people would find it desirable to have the hcloud cli also having the possibility to output a "age" column that would look something like this:

```bash
~/src/cedi/cedi.dev/k8s/k8s-manifests ❯ hcloud server list -s name -o 'columns=name,status,ago,ipv4,ipv6'
NAME                           STATUS    AGE   IPV4             IPV6                   
cedi-dev-control-plane-xxxxx   running   5m    XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-control-plane-xxxxx   running   14m   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-control-plane-xxxxx   running   25m   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-worker-cxp31-xxxxx    running   34m   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-worker-cxp31-xxxxx    running   35m   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-worker-cxp31-xxxxx    running   4d    XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-worker-cxp31-xxxxx    running   4d    XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
```

Since this would also make it much more concise compared to the `CREATED` timestamp, I would also love to see this integrated into the default_columns.

I would also volunteer to implement this myself and submit a PR, but I wanted to gauge interest first, before putting in some work and having the PR rejected.

### PR DESCRIPTION
**Pull request by cedi**
As an SRE, I deploy a lot of hcloud resources automatically via the hcloud API and use the cli to validate those changes. I am deploying using immutable VMs rather than phoenix deployments (instead of updating existing VMs, I re-create them with the newest config) Since I am using the API in automation to create hcloud resources, my hcloud resources include Ids in their name, making it a mess sometimes to figure out which is the newest VM. To know which VM got deployed when, I often find myself using the hcloud-cli like this:

```bash
$ hcloud server list -s name -o 'columns=name,status,created,ipv4,ipv6'
NAME                           STATUS    CREATED                        IPV4             IPV6
cedi-dev-control-plane-xxxxx   running   Sun Nov 13 13:43:05 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-control-plane-xxxxx   running   Sun Nov 13 13:51:46 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-control-plane-xxxxx   running   Sun Nov 13 13:40:22 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-worker-cxp31-xxxxx    running   Wed Nov  9 17:15:32 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-worker-cxp31-xxxxx    running   Wed Nov  9 17:26:36 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-worker-cxp31-xxxxx    running   Wed Nov  9 16:46:55 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
cedi-dev-worker-cxp31-xxxxx    running   Sun Nov 13 14:02:41 CET 2022   XXX.XXX.XXX.XXX  2a01:4f8:xxxx:xxxx::/64
```

However, I noticed the `created` column contains the "raw" create DateTime string which is good for computers to read, but bad for humans. Using the hcloud dashboard in my browser, I can see the created timestamp as a Duration since `time.Now()`:

Screenshot 2022-11-13 at 14 17 18

With this commit I add a "age" column to the output of hcloud cli:

```bash
$ hcloud server list
ID         NAME                           STATUS    IPV4             IPV6                      PRIVATE NET           DATACENTER   AGE
25550867   cedi-dev-control-plane-xxxxx   running   xxx.xxx.xx.xxx   2a01:4f8:xxxx:xxxx::/64   10.0.0.x (cedi-dev)   fsn1-dcxx    2d
25551100   cedi-dev-control-plane-xxxxx   running   xx.xx.xxx.xx     2a01:4f8:xxx:xxxx::/64    10.0.0.x (cedi-dev)   fsn1-dcxx    2d
25551348   cedi-dev-worker-cxp31-xxxxx    running   xx.xx.xx.xxx     2a01:4f8:xxxx:xxxx::/64   10.0.0.x (cedi-dev)   fsn1-dcxx    2d
25586128   cedi-dev-worker-cxp31-xxxxx    running   xx.xx.xxx.xx     2a01:4f8:xxxx:xxxx::/64   10.0.0.x (cedi-dev)   fsn1-dcxx    1d
25586289   cedi-dev-control-plane-xxxxx   running   xxx.xx.xx.xx     2a01:4f8:xxxx:xxxx::/64   10.0.0.x (cedi-dev)   fsn1-dcxx    1d
25588261   cedi-dev-worker-cxp31-xxxxx    running   xxx.xx.xxx.xxx   2a01:4f8:xxx:xxxx::/64    10.0.0.x (cedi-dev)   fsn1-dcxx    23h

$ hcloud load-balancer list
ID       NAME                            IPV4              IPV6                    TYPE   LOCATION   NETWORK ZONE   AGE
912000   cedi-dev-kube-apiserver-xxxxx   xxx.xxx.xxx.xxx   2a01:4f8:xxxx:xx::1     lb11   fsn1       eu-central     24d
912020   cedi-dev-ingress-lb             xxx.xxx.xxx.xx    2a01:4f8:xxxx:xxxx::1   lb11   nbg1       eu-central     24d
```

I also added the "age" column to the "default_columns" in most commands

### CODE REVIEW

**Review by apricote — COMMENTED** [review]
This looks great! It would be awesome if you could add some simple unit tests for the `util.Age` function, but I can also do that after we merge this PR if you don't have the time for it.

**Comment by cedi** [comment]
> This looks great! It would be awesome if you could add some simple unit tests for the `util.Age` function, but I can also do that after we merge this PR if you don't have the time for it.

Hey @apricote, thanks for the feedback! I totally forgot adding unit-testing for this. Sorry!

I added some tests for `Ago`, as well as fixing the existing tests that I accidentally broke

File: internal/cmd/util/util_internal_test.go
Comment (apricote) [inline internal/cmd/util/util_internal_test.go:53]:
```suggestion
func TestAge(t *testing.T) {
```

**Review by apricote — CHANGES_REQUESTED** [review]
Thanks for your effort :)

File: internal/cmd/network/list_test.go
Comment (cedi) [reply internal/cmd/network/list_test.go]: fixed in [1866049](https://github.com/hetznercloud/cli/pull/420/commits/186604952d720488d0fc2c196a78d107080ab006)

**Pull request merged**
